### ISSUE TITLE
[accessibility issue] The geomap has no keyboard accessible way to pan the map

### ISSUE THREAD

**Issue by ckbedwell** [grafana/grafana#79628]
**Steps to reproduce**:
Go any geomap implementation in Grafana. Try to pan the map with just your keyboard.

**Actual Result**:
You are unable to pan the map.

**Expected Result**
There should be a mechanism (such as four direction buttons) which a keyboard user can interact with to pan the map.

UHG Audit Id: 2.11
